README.md

```diff
@@ -82,6 +81,14 @@ dependency and substitute the `make` commands in the instructions below for:
 make TAGS=nolzma
 ```
 
+#### Glib's GIO support opt-out
+
+If no Glib's GIO library support if desired, you can ignore the `glib2-dev` package
+dependency and substitute the `make` commands in the instructions below for:
+
+```
+make TAGS=nolibgio
```

Comment:
Same here, even though it's relevant to mention glib, I think the tag itself should be feature focused, not library focused, so I would call it `nodbus`.